apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -44,49 +44,39 @@ export function Workspace({
       >
         ${RebootModal({ __csrf_token })} ${ResetModal({ __csrf_token })}
 
-        <nav
-          class="navbar navbar-expand-md navbar-dark bg-info align-items-center"
-          style="height:55px"
-        >
+        <nav class="navbar navbar-light navbar-expand-lg bg-info align-items-center">
           <div class="container-fluid">
-            <div class="d-flex flex-column mr-3 text-white">
-              <span>
-                <h1 class="h6 font-weight-normal mb-0">
-                  <a href="${navTitleHref}" target="_blank" class="text-white">${navTitle}</a>
-                </h1>
-              </span>
-              <span class="small">
+            <div class="d-flex flex-column mr-3">
+              <h1 class="h6 font-weight-normal mb-0">
+                <a href="${navTitleHref}" target="_blank" style="color: #000;">${navTitle}</a>
```

Comment:
Just curious, is using `.text-dark` insufficient here (and elsewhere using `style="color: #000"`)?

Reply:
In this case, it is in fact insufficient. `.text-dark` is actually a darkish gray, not black, and this has insufficient contrast on the background used on this page. You can plug the colors into https://webaim.org/resources/contrastchecker/ to check this:

<img width="730" alt="Screenshot 2024-08-19 at 15 52 07" src="https://github.com/user-attachments/assets/a5a58b9f-29c5-4a6f-8c95-5621fca8673c">

Whereas pure black has sufficient contrast (at least for WCAG AA, which is what we're targeting at the moment):

<img width="721" alt="Screenshot 2024-08-19 at 15 52 43" src="https://github.com/user-attachments/assets/4c9ab3f8-25cc-4fca-8faf-e00fc8fc5d45">

With BS5 styles, black in fact passes the WCA AAA contrast requirements too:

<img width="752" alt="Screenshot 2024-08-19 at 15 53 43" src="https://github.com/user-attachments/assets/f911341a-c0de-4902-b252-ceebfd1ba30b">

Reply:
Good to know, thanks!